### DISCUSSION TITLE
使用fallback回落进行多域名https访问请问如何配置

### DISCUSSION THREAD

**Discussion by kettly1260** [XTLS/Xray-core#530] [Q&A]
现在用的是fallback回落到nginx，使用的是一个泛域名
但是我同时还有其它域名需要布置，但是看教程只能通过SNI分流，且域名证书要SAN证书才行。请问有什么办法可以添加多个泛域名证书或者多个域名证书实现网页的正常访问

**Comment by lxhao61** [comment]
如下两种方法之一都可以实现你需求。
1、nginx SNI分流
2、fallbacks SNI shunt
参考https://github.com/lxhao61/integrated-examples

**Reply by kettly1260** [reply]
请问fallback sni shunt是fallback之后加name进行分流吗？这个好像要san证书才可以反正不同顶级域名的网页吧？请问这个具体怎么配置呢

**Comment by kettly1260** [comment]
@lxhao61 请问fallback sni shunt是fallback之后加name进行分流吗？这个好像要san证书才可以反正不同顶级域名的网页吧？请问这个具体怎么配置呢

**Reply by lxhao61** [reply]
是的。不需要，可以并行多个证书。

**Reply by kettly1260** [reply]
"certificates": [
                {
                    "certificateFile": "/path/to/fullchain.crt", // 换成你的证书，绝对路径
                    "keyFile": "/path/to/private.key" // 换成你的私钥，绝对路径
                },
                {
                    "certificateFile": "/path/to/fullchain2.crt", // 换成你的证书，绝对路径
                    "keyFile": "/path/to/private2.key" // 换成你的私钥，绝对路径
                }

请问是这样吗

**Reply by lxhao61** [reply]
是的。

**Reply by kettly1260** [reply]
配置以后可用。谢谢指导
